snappass/templates/expired.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="container">
+  <section>
+    <div class="page-header"><h1>Secret not found, used or expired</h1></div>
```

Comment:
I think it's a little nicer to just use a simple heading:

```suggestion
    <div class="page-header"><h1>Secret not found</h1></div>
```